dist/s_clang_format

```diff
@@ -32,6 +32,10 @@ download_clang_format() {
         curl https://s3.amazonaws.com/boxes.10gen.com/build/build/clang-format-llvm-"$version"-x86_64-apple-darwin.tar.gz -o $archive
         tar --strip=1 -C dist/ -xf $archive clang-format-llvm-"$version"-x86_64-apple-darwin/clang-format && rm $archive
         ;;
+    "arm64-Darwin")
+        curl https://s3.amazonaws.com/boxes.10gen.com/build/build/clang-format-llvm-"$version"-x86_64-apple-darwin.tar.gz -o $archive
```

Comment:
Can we have a comment here explaining that the x86 binary runs on arm with no issues?

Reply:
done